website/pages/docs/k8s/crds.mdx

```diff
@@ -0,0 +1,441 @@
+   expected that `connectInject` will be enabled.
+
+See [Install with Helm Chart](/docs/k8s/installation/install) for further installation
+instructions.
+
+## Usage
+
+Once installed, we can use `kubectl` to create and manage Consul's configuration entries.
+
+### Create
+
+We can create configuration entries via `kubectl apply`.
+
+```shell-session
+$ cat <<EOF | kubectl apply -f -
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceDefaults
+metadata:
+  name: foo
+spec:
+  protocol: "http"
+EOF
+
+servicedefaults.consul.hashicorp.com/foo created
+```
+
+See [Specifications](#specifications) for detailed schema documentation.
+
+### Get
+
+We can use `kubectl get [kind] [name]` to get the status of the configuration entry:
+
+```shell-session
+$ kubectl get servicedefaults foo
+NAME  SYNCED
+foo   True
+```
+
+The `SYNCED` status shows whether the configuration entry was successfully created
+in Consul.
+
+### Describe
+
+We can use `kubectl describe [kind] [name]` to investigate the status of the
+configuration entry. If `SYNCED` is false, the status will contain the reason
+why.
+
+```shell-session
+$ kubectl describe servicedefaults foo
+Status:
+  Conditions:
+    Last Transition Time:  2020-10-09T21:15:50Z
+    Status:                True
+    Type:                  Synced
+```
+
+### Edit
+
+We can use `kubectl edit [kind] [name]` to edit the configuration entry:
+
+```shell
+$ kubectl edit servicedefaults foo
+# change protocol: http => protocol: tcp
+servicedefaults.consul.hashicorp.com/foo edited
+```
+
+We can then use `kubectl get` to ensure the change was synced to Consul:
+
+```shell-session
+$ kubectl get servicedefaults foo
+NAME  SYNCED
+foo   True
+```
+
+### Delete
+
+We can use `kubectl delete [kind] [name]` to delete the configuration entry:
+
+```shell-session
+$ kubectl delete servicedefaults foo
+servicedefaults.consul.hashicorp.com "foo" deleted
+```
+
+We can then use `kubectl get` to ensure the configuration entry was deleted:
+
+```shell-session
+$ kubectl get servicedefaults foo
+Error from server (NotFound): servicedefaults.consul.hashicorp.com "foo" not found
+```
+
+#### Delete Hanging
+
+If running `kubectl delete` hangs without exiting, there may be
+a requirement inside Consul that prevents that configuration entry from being
+deleted. For example, if you set the protocol of your service to `http` via `ServiceDefaults` and then
+create a `ServiceSplitter`, you won't be able to delete the `ServiceDefaults`.
+
+This is because by deleting the `ServiceDefaults` config, you are setting the
+protocol back to the default which is `tcp`. Since `ServiceSplitter` requires
+that the service has an `http` protocol, Consul will not allow the `ServiceDefaults`
+to be deleted since that would put Consul into a broken state.
+
+In order to delete the `ServiceDefaults` config you would need to first delete
+the `ServiceSplitter`.
+
+## Specifications
+
+### ProxyDefaults
```

Comment:
Would it make sense within each of these subsections linking out to the main docs for them like https://www.consul.io/docs/agent/config-entries/service-router